packages/moonstone/Scrollable/ScrollableNative.js

```diff
@@ -546,9 +605,26 @@ class ScrollableNative extends Component {
 									ref: this.initChildRef
 								})}
 							</TouchableDiv>
-							{isVerticalScrollbarVisible ? <Scrollbar {...verticalScrollbarProps} {...this.scrollbarProps} disabled={!isVerticalScrollbarVisible} /> : null}
+							{isVerticalScrollbarVisible ?
```

Comment:
If isVerticalScrollbarVisible is false, do you need to put arialabel in verticalScrollbarA11yProps above?

Reply:
No. There is no DOM element to put aria-label.

Reply:
Do we need to create a variable(horizontalScrollbarA11yProps, verticalScrollbarA11yProps) that does not apply to the DOM element?

horizontalScrollbarA11yProps = {
	nextButtonAriaLabel: horizontalScrollbarNextButtonAriaLabel,
	previousButtonAriaLabel: horizontalScrollbarPreviousButtonAriaLabel
},
verticalScrollbarA11yProps = {
	nextButtonAriaLabel: verticalScrollbarNextButtonAriaLabel,
	previousButtonAriaLabel: verticalScrollbarPreviousButtonAriaLabel
};

Reply:
`isVerticalScrollbarVisible` and `isHorizontalScrollbarVisible` variables were passed as ui `Scrollable`'s `containerRenderer` parameters. That means that we need to define `horizontalScrollbarA11yProps`, `verticalScrollbarA11yProps` in  containerRenderer(). In my opinion, it would be better to define them as I did then to define them in containerRenderer().